src/Abc.Zebus/Dispatch/MessageDispatcher.cs

```diff
@@ -237,6 +251,7 @@ internal enum MessageDispatcherStatus
     {
         Stopped,
         Stopping,
-        Started
+        Started,
+        StartedDeliveringMessages
```

Comment:
I'd rename `Started` to `Starting` and `StartedDeliveringMessages` to `Started`

Reply:
I agree that it might make it easier to understand.  I liked the symmetry with the events on IBus but maybe it's not worth it...